src/index_create_aws_iam_roles.js

```diff
@@ -94,6 +95,16 @@ exports.handler = function(event, context, callback) {
       awsIamRole.createRole(payload, (err, data) => {
         console.log('Error:', err);
         console.log('Res:', data);
+        if (roles[i].roleName === process.env.SC_ADMIN_ROLE_NAME){
+          for (let k = 0; k < crawspolicies.length; k++){
+             options.policyArn = crawspolicies[k].policyArn;
+             options.roleName = process.env.SC_ADMIN_ROLE_NAME;
+             awsIamRole.attachRolePolicy(options, (err, res) => {
```

Comment:
I am a bit reluctant to make those many calls to AWS, we should be creating a customer managed policy document and use it once to attach to the role.

Read about creating customer managed policy and prepare an inline policy document instead of using these managed policies. Refer, https://gist.github.com/bernadinm/6f68bfdd015b3f3e0a17b2f00c9ea3f8 which help you know the policy document for AWS managed policies.

Reply:
Not very clear with the ask.
Do you want me to create one inline policy document and attach all the permissions of those 5 managed policies into that document and attach to the role? That will then become similar to a inline policy. 
But AWS always recommends using managed policies instead of inline policies.

Reply:
What was the policy attached to SC_ADMIN_ROLE_NAME before?